### PR TITLE
feat(media): add announcements and feature tour for media enhancements

### PR DESCRIPTION
**Pull request by dcshzj**
**NOTE: This will only go in once we have the complete flow available**

## Problem

Closes IS-691 and IS-692.

## Solution

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
- [x] No - this PR is backwards compatible with ALL of the following feature flags in this [doc](https://www.notion.so/opengov/Existing-feature-flags-518ad2cdc325420893a105e88c432be5)

**Features**:

- The announcements modal and feature tour has been added.

## Tests

- [ ] Unit tests (using `npm run tests`)
- [ ] e2e tests (comment on this PR with the text `!run e2e`)
- [x] Smoke tests
    - [ ] Log in as a GitHub user
    - [ ] Verify that the announcement modal shows with the newest announcement about the media enhancements
    - [ ] Access any site's image workspace
    - [ ] Verify that you can trigger the feature tour
    - [ ] Select any image
    - [ ] Verify that the selected feature tour can be triggered

## Deploy Notes

*None*

### CODE REVIEW

**Comment by dcshzj** [comment]
!run chromatic

File: src/layouts/Media/Media.tsx [hunk 1]
```diff
@@ -264,7 +286,10 @@ export const Media = (): JSX.Element => {
                         Deselect all
                       </Button>
                     </Box>
-                    <Box mt="auto">
+                    <Box
+                      mt="auto"
+                      id="isomer-media-onselect-feature-tour-step-1"
```
Comment (seaerchin) [inline src/layouts/Media/Media.tsx:291]: for the ids here (and below), could we use `MEDIA_ONSELECT_FEATURE_STEPS.target`?

**Review by seaerchin — APPROVED** [review]
**please only merge after addressing comment**

File: src/layouts/Media/Media.tsx (hunk 1 above)
Comment (dcshzj) [reply src/layouts/Media/Media.tsx]: Fixed in 85f50571.

**Pull request merged**